packages/m/materialstyle.json

```
@@ -0,0 +1,51 @@
+{
+  "description": "Material Style is a Material Design UI library based on Bootstrap.",
+  "filename": "js/materialstyle.min.js",
+  "homepage": "https://materialstyle.github.io/",
+  "keywords": [
+    "materialstyle",
+    "material-style",
+    "material-design",
+    "css",
+    "sass",
+    "scss",
+    "mobile-first",
+    "responsive",
+    "front-end",
+    "framework",
+    "web",
+    "bootstrap",
+    "bootstrap-material-design",
+    "bootstrap-theme",
+    "google-material-design"
+  ],
+  "license": "MIT",
+  "name": "@materialstyle/materialstyle",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/materialstyle/materialstyle"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "@materialstyle/materialstyle",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*.@(js|css|map)"
```

```suggestion
          "js/*.js?(.map)",
          "css/*.css?(.map)"
```
I don't think we need a glob-star here (it can lead to files being included that we don't want). The package seems to have all the CSS directly in `dist/css`, and all the JS directly in `dist/js`